.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Synopsys
+
+on:
+  push:
+    branches: [coverity_scan]
```

Comment:
Is it a leftover?

Reply:
We are limited to a few scans per week/day:
[What is the frequency for build submissions to Coverity Scan?](https://scan.coverity.com/faq#frequency)

We have been checking only the `coverity_scan` branch, which we update manually.

Reply:
Ah OK, I missed that. Perfect then!